<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Name</class>
 <widget class="QWidget" name="Name">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Name{
background-color: rgb(255, 240, 245);
}

QPushButton{
border-radius:8px;
border: 2px solid rgb(163, 137, 131);
background-color: rgb(196, 170, 164);
	font: 75 18pt &quot;Times New Roman&quot;;
}
QPushButton::hover{
border: 2px solid rgb(163, 137, 131);
background-color: rgb(214, 186, 180);
}
QLabel{
font: 75 18pt &quot;Times New Roman&quot;;
/*background-color: rgb(212, 188, 181);*/
background-color: rgb(236, 215, 210);
border: 1px solid rgb(163, 137, 131);
}
QLineEdit{
font: 75 18pt &quot;Times New Roman&quot;;
border: 2px solid rgb(163, 137, 131);
}
QGroupBox{
border-radius:10px;
border: 2px solid rgb(163, 137, 131);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>801</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите Имя и Фамилию:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите номер варианта :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
